docs/docs/platform/subscribers.md

```diff
@@ -33,12 +33,11 @@ Novu will create a subscriber if one does not exist, and will update existing su
 
 ### Subscriber identifier
 
-This is a unique identifier used by Novu to keep track of a specific subscriber. We recommend using the internal id your application uses for a specific user.
-Using an identifier like email might cause issues locating a specific subscriber once they change their email address.
+This is a unique identifier used by Novu to keep track of a specific subscriber. We recommend that you use the internal id that your application uses for a specific user. Using an identifier like email might cause issues locating a specific subscriber once they change their email address.
```

Comment:
I consider the existing copy is clearer than the suggested one and it is correct grammatically. Also the suggested copy is not bringing any added value. Let's keep the original, please.

Reply:
"internal id your application u....." this had issues while reading

Reply:
@p-fernandez even when you read this sentence "Data stored on the subscriber object that allows you to easily access it in the notification templates you create.. "this is also not correct and i changed it to "Data stored in the subscriber object that you can easily access in your notification templates

Reply:
if u want i will update this as it is "This is a unique identifier used by Novu to keep track of a specific subscriber. We recommend using the internal id your application uses for a specific user.
Using an identifier like email might cause issues locating a specific subscriber once they change their email address."

Reply:
> "internal id your application u....." this had issues while reading

I kindly disagree as stated in my original message.

Reply:
ok i will keep it as it is then

Reply:
> @p-fernandez even when you read this sentence "Data stored on the subscriber object that allows you to easily access it in the notification templates you create....." this is also not correct and i changed it to "Data stored in the subscriber object that you can easily access in your notification templates

I am sorry but this comment has nothing to do with the copy the rollback has been requested.